data/mods/Medieval_Stuff/tools.json

```diff
@@ -0,0 +1,543 @@
+    "cutting": 2,
+    "to_hit": 2,
+    "max_charges": 0,
+    "initial_charges": 0,
+    "charges_per_use": 0,
+    "turns_per_charge": 0,
+    "ammo": "NULL",
+    "revert_to": "null",
+    "use_action": [
+        {
+            "type": "auto_transform",
+            "msg": "You shift to a one-handed stance.",
+            "target": "longsword_one_fake",
+            "active": false,
+            "menu_option_text": "Shift stance"
+        }
+    ],
+    "category" : "weapons"
+  },
+  {
+    "id": "estoc_one_fake",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "estoc (one-handed stance)",
+    "name_plural": "estocs (one-handed stance)",
+    "description": "This is a dull, cheaply-made replica of a thin sword resembling a longsword.  You can still switch stances with it, if you want.",
+    "price": 13500,
+    "material": "aluminum",
+    "techniques": ["RAPID", "WBLOCK_2", "PRECISE"],
+    "flags" : ["BASH", "SHEATH_SWORD"],
+    "weight": 939,
+    "volume": 10,
+    "bashing": 2,
+    "cutting": 2,
+    "to_hit": 2,
+    "max_charges": 0,
+    "initial_charges": 0,
+    "charges_per_use": 0,
+    "turns_per_charge": 0,
+    "ammo": "NULL",
+    "revert_to": "null",
+    "use_action": [
+        {
+            "type": "auto_transform",
+            "msg": "You shift to a two-handed stance.",
+            "target": "estoc_two_fake",
+            "active": false,
+            "menu_option_text": "Shift stance"
+        }
+    ],
+    "category" : "weapons"
+  },
+  {
+    "id": "estoc_two_fake",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "estoc (two-handed stance)",
+    "name_plural": "estocs (two-handed stance)",
+    "description": "This is a dull, cheaply-made replica of a thin sword resembling a longsword.  You can still switch stances with it, if you want.",
+    "price": 13500,
+    "material": "aluminum",
+    "techniques": ["WBLOCK_2", "PRECISE"],
+    "flags" : ["BASH", "SHEATH_SWORD", "ALWAYS_TWOHAND"],
+    "weight": 939,
+    "volume": 10,
+    "bashing": 3,
+    "cutting": 2,
+    "to_hit": 2,
+    "max_charges": 0,
+    "initial_charges": 0,
+    "charges_per_use": 0,
+    "turns_per_charge": 0,
+    "ammo": "NULL",
+    "revert_to": "null",
+    "use_action": [
+        {
+            "type": "auto_transform",
+            "msg": "You shift to a one-handed stance.",
+            "target": "estoc_one_fake",
+            "active": false,
+            "menu_option_text": "Shift stance"
+        }
+    ],
+    "category" : "weapons"
+  },
+  {
+    "id": "halberd",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "halberd",
+    "description": "This is a versatile polearm with an axe blade, a spike, and other fun things attached to a long stick.",
+    "price": 50000,
+    "material": ["wood", "steel"],
+    "flags": ["CHOP", "DURABLE_MELEE", "REACH_ATTACK"],
+    "techniques": ["WBLOCK_1", "WIDE", "IMPALE"],
```

Comment:
Halberd shouldn't impale. Impaling strike refers to attacking 2 opponents in a row. The axe blade on halberd prevents that.